wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -175,53 +182,58 @@ getHistoryLimited mCWalId mAccId mAddrId mSkip mLimit = do
         "Please do not specify both walletId and accountId at the same time"
 
 addHistoryTxMeta
-    :: MonadWalletDB ctx m
-    => CId Wal
+    :: (MonadIO m, HasConfiguration)
+    => WalletDB
+    -> CId Wal
```

Comment:
Nice, ideally that's the type of monad we want to work in for non-toplevel functions 😀